rustler_tests/test/binary_test.exs

```diff
@@ -3,7 +3,7 @@ defmodule RustlerTest.BinaryTest do
 
   test "subbinary creation" do
     assert RustlerTest.make_shorter_subbinary("test") == "es"
-    assert_raise ErlangError, fn -> RustlerTest.make_shorter_subbinary("t") end
```

Comment:
@hansihe the assertion tested that the function `make_shorter_subbinary()` panics due to a `usize` underflow. With profile `release`, this panic does not happen any longer: the `length` wraps around, causing `make_subbinary()` to return an error. Is this OK, or should the function `make_shorter_subbinary()` panic (unwrap on `checked_sub()`)?